yarn.lock

```diff
@@ -3243,9 +3243,9 @@ ember-paper@^1.0.0-beta.4:
     resolve "^1.3.3"
     virtual-each "~0.6.0"
 
-ember-power-select@2.0.8:
-  version "2.0.8"
-  resolved "https://registry.yarnpkg.com/ember-power-select/-/ember-power-select-2.0.8.tgz#1d3ed341ef2c620df3c2b0fb31c5d951382d3c06"
+ember-power-select@2.0.9:
```

Comment:
Somebody failed in uploading the yarn.lock? Because in the package it was 2.0.9 already